### ISSUE TITLE
1-3 resume review sentence guidance: add example of shorter justification for RAT use

### ISSUE THREAD

**Issue by stephaniefaith** [usds/SME-QA-Website#168]
We need to add a second set of instructions around resume review for all the use cases where they aren't inputting sentences in a blank field. Both the RAT and EPA's excel automatically have drop downs or radio buttons that tell you pass/fail on the requirements and therefore we need to show what the justification statement should look like in those cases. 

- [x] add this to the assessment strategy template doc

- [x] add this to the RR training PPT

- [ ] third party review - Will?

**Comment by stephaniefaith** [comment]
[sme-training-resume-review.pptx](https://github.com/usds/SME-QA-Website/files/4340086/sme-training-resume-review.pptx)
[sme-training-resume-review multirole.pptx](https://github.com/usds/SME-QA-Website/files/4340089/sme-training-resume-review.multirole.pptx)

**Comment by stephaniefaith** [comment]
this is already in the existing job analysis template: Rather than writing sentences from scratch, this hiring action may make use of an SME Resume Review tool which has all technical specialized experience listed in a checkbox format. Once an SME says one is missing, they do not need to complete the rest of the checklist. If an SME says one is missing they will be prompted to write a sentence or two explaining why the competency or proficiency level was missing.

**Comment by wslack** [comment]
@stephaniefaith where is this discussed in the new PPT draft? I read through yours and it doesn't seem to have content about the RAT.

**Comment by stephaniefaith** [comment]
@wslack let's review friday

**Comment by wslack** [comment]
Also edit this page: https://smeqa.usds.gov/toolkit/reviewing-resumes/resume-review-training-agenda/

**Comment by wslack** [comment]
Closing as OBE
